src/devices/cpu/z80/z80.lst

```diff
@@ -728,13 +741,12 @@ ffff
 		debugger_wait_hook();
 		call rop
 		PC--;
-		continue;
+		goto rop;
 	} else {
 		PRVPC = PC;
 		debugger_instruction_hook(PC);
 		call rop
-		m_ref = (0x00 << 16) | (TDAT8 << 8);
-		goto process;
+		goto *opc_label[0x0400 | TDAT8];
```

Comment:
Is this portable?  I thought these computed `goto` statements were a non-standard language extension.  Don’t we prefer to at least have a standard-conformant fallback for compiler-specific things?

Reply:
You could make another PR with the corrections I noticed, like nsc/ncs typo, and the double t6a84 in the lua file

Reply:
btw what would be the condition for this?